app/cdap/components/CaskWizards/AddNamespace/index.js

```diff
@@ -112,21 +112,55 @@ export default class AddNamespaceWizard extends Component {
       namespaceId: newNamespaceId,
     });
     let buttonLabel = T.translate(`${PREFIX}.callToAction`, { namespaceId: newNamespaceId });
-    let linkLabel = T.translate('features.Wizard.GoToHomePage');
     this.setState({
       successInfo: {
         message,
         buttonLabel,
         buttonUrl: window.getAbsUIUrl({
           namespaceId: newNamespaceId,
         }),
-        linkLabel,
-        linkUrl: `${window.getAbsUIUrl({
-          namespaceId: currentNamespaceId,
-        })}/control`,
+        links: this.getLinks(newNamespaceId, currentNamespaceId),
       },
     });
   }
+
+  isNamespacedServiceAccountsEnabled() {
+    return window.CDAP_CONFIG.featureFlags['feature.namespaced.service.accounts.enabled']
+      ? true
+      : false;
```

Comment:
`Boolean(window.CDAP_CONFIG.featureFlags['feature.namespaced.service.accounts.enabled'])`

Reply:
`!!` is also acceptable: https://google.github.io/styleguide/tsguide.html#type-coercion

Reply:
Is `window.CDAP_CONFIG.featureFlags['feature.namespaced.service.accounts.enabled']` a string or a boolean here? `Boolean('false')` evaluates to `true`. If it's already a boolean, `!!` would be sufficient.

Reply:
Yes @njbriggs , you are right `window.CDAP_CONFIG.featureFlags['feature.namespaced.service.accounts.enabled']` is returning string values of 'true' \ 'false'. 

@GnsP :  I think we need to check for strings and return the value here. what do u suggest?

Reply:
Yeah, the feature flags are string ("true" / "false") or undefined. Casting it to boolean can result in incorrect value. In this case, I think `window.CDAP_CONFIG.featureFlags['feature.namespaced.service.accounts.enabled'] === "true"` should be a cleaner solution.